.github/workflows/ci.yaml

```diff
@@ -1026,6 +1028,44 @@ jobs:
         if: ${{ github.repository_owner == 'coder' && github.ref == 'refs/heads/main' }}
         run: rm -f /tmp/{apple_cert.p12,apple_cert_password.txt,apple_apikey.p8}
 
+  check-build:
+    # This job runs make build to verify compilation on PRs
```

Comment:
Add to the comment that the build doesn't get signed and is not suitable for actual usage.